src/components/Reference/Reference.tsx

```diff
@@ -70,7 +70,7 @@ function Canvas() {
         if (crop.scale >= 1 && crop.scale <= 5)
           setCrop((crop) => ({
             ...crop,
-            scale: crop.scale + event.deltaY * 0.0001,
+            scale: crop.scale - event.deltaY * 0.0001,
```

Comment:
I feel like increasing this constant maybe to 0.001 instead of 0.0001 would make the zoom speed a bit faster.

Also, having something like:
```
const newScale = Math.min(
  5,
  Math.max(1, crop.scale - event.deltaY * 0.001),
);
setCrop((crop) => ({
  ...crop,
  scale: newScale,
}));
```
would remove the need for lines 75 and 76, and would also solve the problem of the zoom flickering if you keep trying to zoom in when scale == 5